lib/order.rb

```diff
@@ -0,0 +1,57 @@
+# require_relative 'sms'
+
+class Order
+
+  DEFAULT_QUANTITY = 1
+
+  attr_reader :basket
+
+  def initialize
+    @basket = {}
+    # @sms_client = sms_client
+  end
+
+  def add(dish, quantity = DEFAULT_QUANTITY)
+    quantity = @basket[dish] + quantity if @basket[dish]
+    @basket[dish] = quantity
+  end
+
+  def remove(dish)
+    fail 'Dish not in basket' if !@basket[dish]
```

Comment:
Style/NegatedIf: Favor `unless` over `if` for negative conditions. (https://github.com/bbatsov/ruby-style-guide#unless-for-negatives)